bedrock-devnet/devnet/__init__.py

```diff
@@ -342,27 +299,34 @@ def devnet_deploy(paths, args):
     log.info(f'Using L2OutputOracle {l2_output_oracle}')
     log.info(f'Using DisputeGameFactory {dispute_game_factory}')
     log.info(f'Using batch inbox {batch_inbox_address}')
-
-    log.info('Bringing up `op-node`, `op-proposer` and `op-batcher`.')
-    run_command(['docker', 'compose', 'up', '-d', 'op-node', 'op-proposer', 'op-batcher'], cwd=paths.ops_bedrock_dir, env={
+    # Set up the base docker environment.
+    docker_env={
         'PWD': paths.ops_bedrock_dir,
-        'L2OO_ADDRESS': l2_output_oracle,
         'SEQUENCER_BATCH_INBOX_ADDRESS': batch_inbox_address,
+        'L2OO_ADDRESS': l2_output_oracle,
```

Comment:
Maybe we don't need 'L2OO_ADDRESS' here. We have another check at: https://github.com/tokamak-network/tokamak-thanos/blob/feat/OR-1619-devnet-ecotone/bedrock-devnet/devnet/__init__.py#L321

Reply:
That's true, updated